spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1148,6 +1148,56 @@
             error_type: Faraday::NilStatusError,
           )
         end
+
+        context 'when a user was flagged with fraud_pending_reason' do
+          before(:each) do
+            pending_enrollment.profile.update!(fraud_pending_reason: 'threatmetrix_review')
+            stub_request_passed_proofing_results
+          end
+
+          it 'logs something useful' do
+            job.perform(Time.zone.now)
+
+            expect(job_analytics).to have_logged_event(
+              'GetUspsProofingResultsJob: User transitioned to fraud_review',
+              hash_including(
+                enrollment_code: pending_enrollment.enrollment_code,
+              ),
+            )
+          end
+
+          it 'does update the enrollment' do
+            freeze_time do
+              expect(pending_enrollment.status).to eq 'pending'
+              job.perform(Time.zone.now)
+              expect(pending_enrollment.reload.status).to eq 'passed'
+            end
+          end
+
+          it 'does not send a success email' do
+            user = pending_enrollment.user
+
+            freeze_time do
+              expect do
+                job.perform(Time.zone.now)
+              end.not_to have_enqueued_mail(UserMailer, :in_person_verified).with(
+                params: { user: user, email_address: user.email_addresses.first },
+                args: [{ enrollment: pending_enrollment }],
+              )
+            end
+          end
+
+          it 'marks the user as fraud_review_pending_at' do
+            freeze_time do
+              job.perform(Time.zone.now)
+
+              pending_enrollment.reload
+              profile = pending_enrollment.profile
+              expect(profile).to be_fraud_review_pending
+              expect(profile).not_to be_active
+            end
```

Comment:
The AC may have changed. You and I talked about exiting behavior vs what was in the issue. I would have expected fraud_review_pending to be null and fraud_review_pending_at to have a timestamp on in.  I may have missed the ask here and/or the requirements/AC have changed upon diving into the code and discovering existing processes. If they have changed- we can update the AC in the ticket.

Reply:
Thanks for flagging this. It did change; there is now a link in the Jira ticket to the Slack thread where it came up. It turns out that the existing pattern is to leave `fraud_pending_reason` in place.

I tried to be clever with `expect(profile).to be_fraud_review_pending?`, which exercises this method:

```ruby
  def fraud_review_pending?
    fraud_review_pending_at.present?
  end
```

But in hindsight I think it might be clearer if I just did `expect(profile.fraud_review_pending_at).to eq (Time.zone.now)` (or similar), rather than relying on a magic matcher + non-obvious helper method in Profile.